modules/ROOT/pages/munit-scaffold-test-task.adoc

```diff
@@ -3,10 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To automatically generate an MUnit test suite from an API definition, you need to have a project generated using APIkit:
+You can use APIKit to automatically generate an empty structure for an MUnit test suite based on your API specification.
 
-. Find your APIKit Router in your project and right click on it.
+This process does not generate a functional implementation of a test. It is meant to make it easier for you to start writing an MUnit test.
+
+Your Mule project must be generated using APIKit.
```

Comment:
I don't understand how generating your Mule "project" relates to generating a test or using APIKit to start writing the test. Can you clarify?
Maybe this info about the project belongs before the specifics of writing the test?
Or maybe the steps need a specific intro so they are not confused with generating the project?